server/integration/testsuite/src/test/java/org/infinispan/server/test/client/rest/RESTOffHeapClientIT.java

```diff
@@ -0,0 +1,51 @@
+package org.infinispan.server.test.client.rest;
+
+import org.infinispan.arquillian.core.InfinispanResource;
+import org.infinispan.arquillian.core.RemoteInfinispanServer;
+import org.infinispan.server.test.category.RESTSingleNode;
+import org.jboss.arquillian.junit.Arquillian;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+
+/**
+ * @since 9.2
```

Comment:
Empty stuff to be removed.

BTW, how about doing Off-Heap tests on functional level in REST module? Perhaps those tests would be faster?

Reply:
The PR has ```RestOperationsOffHeapTest``` for that.  IMO we need both, since usually the behavior of embedded servers is different

Reply:
"since 9.2" is not empty, it's important information about when the test was introduced :smile:

Reply:
@gustavonalle why not keep the `@since` tag and also add a short description? ;)

Reply:
@danberindei IMO the test name is self explanatory, WDYT?